src/manhole/__init__.py

```diff
@@ -400,7 +400,7 @@ def __call__(self, message):
                     os.write(self.destination, full_message.encode('ascii', 'ignore'))
                 else:
                     self.destination.write(full_message)
-            except:  # pylint: disable=W0702
+            except Exception:  # pylint: disable=W0702
```

Comment:
We can remove now the # pylint: comment:
http://pylint-messages.wikidot.com/messages:w0702

Is this related to the eventlet fix?

Reply:
Not really but `tox -e check` was complaining.